config/http.conf

```diff
@@ -2,12 +2,19 @@ http {
   include /etc/nginx/mime.types;
   include /etc/nginx/log.conf;
 
-  server_tokens         off;
-  keepalive_timeout     20s;
-  sendfile              on;
-  tcp_nopush            on;
-  client_max_body_size  400m;
-  client_body_timeout   300s;
+  server_tokens           off;
+  sendfile                on;
+  tcp_nopush              on;
+
+  client_max_body_size    500m;
+  client_body_buffer_size 128k;
+  client_body_timeout     300s;
+  client_header_timeout   605s;
+  keepalive_timeout       605s;
+  proxy_connect_timeout   60s;
+  proxy_read_timeout      600s;
+  proxy_send_timeout      60s;
+  send_timeout            60s;
```

Comment:
Are we setting these to their defaults just for visibility?

Reply:
Yes. For visibility.